docs/management/managing-fields.asciidoc

```diff
@@ -78,6 +78,8 @@ include::field-formatters/color-formatter.asciidoc[]
 
 [[scripted-fields]]
 === Scripted fields
+deprecated:[7.13.0,Scripted fields are deprecated] Use {ref}/runtime.html[runtime fields] instead of scripted fields. Runtime fields
```

Comment:
As the deprecation applies to the entire section, how about using the banner format instead:

deprecated::[7.13.0,Use {ref}/runtime.html[runtime fields] instead of scripted fields. Runtime fields support Painless scripts and provide greater flexibility.]